annotations/management/commands/import_tenses.py

```diff
@@ -30,59 +30,78 @@ def handle(self, *args, **options):
 
 
 def process_file(f, language, use_other_label, model='annotation'):
-    for n, row in enumerate(f):
-        if n == 0:
-            continue
-
-        row = row.strip()
+    f = iter(f)
+    header = next(f)
+    if isinstance(header, bytes):
+        header = header.decode()
+    header = header.strip().split('\t')
+
+    columns = []
+    for h in header[1:]:
+        if isinstance(h, bytes):
+            h = h.decode()
+        if h.lower() in ['tense', 'comments']:
+            columns.append(h)
+        else:
+            try:
+                key = LabelKey.objects.get(title__iexact=h)
+                columns.append(key)
+            except LabelKey.DoesNotExist:
+                raise ValueError('Unknown label "{}"'.format(h))
+
+    for row in f:
+        row = row.decode().strip()
         if row:
-            encoded = [c.decode('utf-8') for c in row.split('\t')]
+            encoded = row.split('\t')
 
             if model == 'annotation':
-                update_annotation(language, encoded, use_other_label)
+                obj = get_annotation(encoded)
             elif model == 'fragment':
-                update_fragment(language, encoded, use_other_label)
+                obj = get_fragment(language, encoded)
```

Comment:
I'm not sure why I originally included language here to look up the Fragment. Could you remove this?